tests/scripts/all.sh

```diff
@@ -3105,7 +3111,7 @@ run_component () {
     local dd_cmd
     dd_cmd=(dd if=/dev/urandom of=./tests/seedfile bs=64 count=1)
     case $OSTYPE in
-        linux*|freebsd*|openbsd*|darwin*) dd_cmd+=(status=none)
+        linux*|freebsd*|openbsd*) dd_cmd+=(status=none)
```

Comment:
Why remove the darwin?

Reply:
This is in a separate commit from the SHA-512 acceleration, and fixes the error

`dd: unknown operand status`

that I was getting when running `all.sh` on macOS

Reply:
Got it.